docs/install/installing-istio.md

```diff
@@ -261,6 +261,54 @@ kubectl apply -f https://raw.githubusercontent.com/knative/serving/master/third_
 **Note:** This method is only for development purposes. The production readiness of the above 
```

Comment:
No need to overstate, use something more succinct, e.g.

**Note:** This installation method is not production ready and is for development purposes only. For a more stable method, use Helm. For more information about the Helm installation method, see [link to helm docs].